### DISCUSSION TITLE
デザインどうすんの問題

### DISCUSSION THREAD

**Discussion by TwoSquirrels** [uchikoshi-fes/uchikoshi-fes.jp-2022#105] [General]
#88 など

テーマは「和」

Web デザイン的には「和モダン」がやりやすそう

**Comment by TwoSquirrels** [comment]
参考になりそうなもの: http://www.kyoto-web.jp/works/index.html

このサイト自体も結構いいのよね: http://www.kyoto-web.jp/

**Reply by TwoSquirrels** [reply]
こんな感じの背景がいい
パーリンノイズで作れそう

**Reply by TwoSquirrels** [reply]
SVG にパーリンノイズ機能があるらしい

**Reply by TwoSquirrels** [reply]
やめたほうがよさそう

**Comment by TwoSquirrels** [comment]
ファーストビューどうしよ #4

**Reply by TwoSquirrels** [reply]
PC では動画にしたいけどその動画どうすんの問題

**Reply by TwoSquirrels** [reply]
部門員より、
こんなの https://www.youtube.com/watch?v=59fZNTuxB-k や
こんなの https://www.youtube.com/watch?v=0KFPtliRn_4 
みたいにしたいと

**Reply by TwoSquirrels** [reply]
とりあえず PV 動画にしてみる

**Comment by TwoSquirrels** [comment]
色は基本
https://www.colordic.org/w

**Reply by TwoSquirrels** [reply]
https://github.com/uchikoshi-fes/uchikoshi-fes.jp-2022/issues/88#issuecomment-1192087961

**Comment by gendaineko2222** [comment]
白と黒を基調としたデザインの中に和風な色のボタンとかタイル型のグリッドレイアウト？を入れるのが手早く和モダンな感じが出るんじゃないかなとイメージしてます
(りすりすさんの全体的な完成イメージがあれば教えて欲しいです)
↑だいたいここに書いてあった()
すみません
